book/scheduling.md

```diff
@@ -307,6 +309,9 @@ class TaskRunner:
         self.condition.release()
 
     def run(self):
+```
+
+``` {.python indent=4 replace=%3d%3d%200/%3d%3d%200%20or%20self.needs_quit}
```

Comment:
Ooh, this is pretty ugly, and I'm worried it won't work nicely with whatever tab guide or don't-split-codeblocks-over-pages solution we come up with. Can we not do this one?